src/ontology/mondo.Makefile

```diff
@@ -908,6 +908,22 @@ config/exclusion_reasons.tsv:
 
 all: config/exclusion_reasons.tsv
 
+##################################
+#### SPARQL Curation pipeline ####
+##################################
+
+tmp/mondo-curation-inferred.ttl:
+	$(ROBOT) convert -I $(OBOBASE)/mondo/mondo-base.owl -f ttl -o $@
```

Comment:
Comment the above two lines out to ensure they are not run.

Reply:
or better, delete them.

Reply:
Done!